lib/noun-project-api/icon_retriever.rb

```diff
@@ -1,14 +1,16 @@
 require 'noun-project-api/retriever'
 
 module NounProjectApi
+  # Retrieve an icon.
   class IconRetriever < Retriever
-    API_PATH = "/icon/"
+    API_PATH = '/icon/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.